pkg/onedir.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# onedir.sh
+#
+# Script for making a onedir build. Requires Docker
+#
+#
+set +e
+
+# XXX: centos 7 with dependencies installed, we need to figure out how we'll
+# handle this
+[ -z "$CONTAINER_IMAGE" ] && CONTAINER_IMAGE=dwoz1/cicd:onedir-centos7
```

Comment:
Should these containers not exist in a saltstack docker hub rather than a personal docker hub account?

Reply:
Yes, any docker images that are being used here will eventually be moved under the saltstack org. 

The repo for the current containers is here: https://github.com/dwoz/docker-dwoz1